cogs/eventstats.py

```diff
@@ -55,10 +55,16 @@ def __init__(self, bot):
                 )
 
         if not ctx.config:
-            # TODO Consider pulling most recent event and if time is between end of event and end of season, show stats.
-            return await ctx.send(
-                'It would appear that you aren\'t currently in an event. Try passing in an Event ID.'
-            )
+            query = "SELECT MAX(id) FROM events WHERE guild_id = $1 AND finish < NOW()"
+            row = await ctx.db.fetchrow(query, ctx.guild.id)
+            if row:
+                event_id = row[0]
+                ctx.config = await self.bot.utils.event_config_id(event_id)
+            else:
+                return await ctx.send(
+                    'It would appear that there are no events connected with this server. Please use `+add event` to '
+                    'create an event or try `+season stats attacks` to see results for the season.'
```

Comment:
Pretty sure it's `+seasonstats attacks`. We could even just direct people to `+stats attacks` since that will divert to `+seasonstats attacks`. 

I'm not even sure if I like the name `+seasonstats` and `+eventstats` - whether `+stats season attacks` and `+stats event attacks` would make more sense, I'm not sure. Anyways we'll leave it like this for now I think.

Reply:
fixed